src/Workspaces/Core/Portable/CodeGeneration/Symbols/CodeGenerationAbstractNamedTypeSymbol.cs

```diff
@@ -110,5 +110,6 @@ public override string MetadataName
 
         public bool IsRefLikeType => Modifiers.IsRef;
 
+        public bool IsNativeInt => false;
```

Comment:
This will probably need further updating -- if you have one of these, pass it to a method that doesn't exist, and invoke Generate Method, what happens?